src/components/SystemActions/CancelAll.vue

```diff
@@ -0,0 +1,170 @@
+<script>
+import { mapActions, mapGetters } from 'vuex'
+
+export default {
+  props: {
+    flowRuns: {
+      type: Array,
+      required: false,
+      default: () => []
+    }
+  },
+  data() {
+    return {
+      finished: false,
+      errors: 0,
+      loaded: 0,
+      loadingKey: 0
+    }
+  },
+  computed: {
+    ...mapGetters('tenant', ['tenant']),
+    loading() {
+      return this.loadingKey > 0
+    }
+  },
+  methods: {
+    ...mapActions('alert', ['setAlert']),
+    async cancelAll() {
+      if (!this.flowRuns?.length) return
```

Comment:
Possibly connected to https://github.com/PrefectHQ/cloud/issues/3429 - this doesn't work but also doesn't show an error in my tenant with 1000+ late runs (which is admittedly rare but we've seen it happen).  Would it be possible to batch by 500 runs?

Reply:
Should we add the same logic to the clear button in the submitted/late runs tile of the flow page?  With this set up one 'clear' button would clear runs while the other (admittedly with a warning dialog) cancels them.

Reply:
We can definitely batch though if I'm reading that issue correctly that wouldn't be the cause of this not working; we don't toggle the schedule here to clear these runs, we pass the runs explicitly and loop through them calling the `delete_flow_run` mutation (which is what @cicdw suggests there). However we _do_ still toggle the schedule on the flow page upcoming runs tile, iirc.

I was holding off on making unnecessary changes to the flow page tile to make this easier to review but I'm fine with switching it so they're consistent 👍🏻

Reply:
Once [bulk deletion](https://github.com/PrefectHQ/cloud/pull/3456) is ready we can use that for Cloud, that'll make this far more performant - I'll wait to update this PR though

Reply:
@znicholasbrown I didn't expose the bulk delete route as a GraphQL call -- is that something you're expecting / need?

Reply:
Oh my bad I read your PR wrong - that would certainly improve our ability to perform bulk actions like this (calling `delete_flow_run` in a loop)

Reply:
Flow page tile has been updated for consistency; I can't seem to recreate the issue with the clear button not working so maybe it's best to leave the batching question to a different PR? (this logic uses the same methods as existing, minus schedule toggling)

Reply:
Sounds ok to me.  One to keep an eye on.  (I think we introduced schedule toggling because the cancel all mutation frequently timed out.)